mlflow/R/mlflow/R/tracking-utils.R

```diff
@@ -140,9 +140,17 @@ parse_run <- function(r) {
   new_mlflow_run(info)
 }
 
+fill_missing_run_cols <- function(r) {
+  expected_names <- c("run_uuid", "experiment_id", "user_id", "status", "start_time",
```

Comment:
I am not sure if this is necessary. If we want to keep it we should have a test to make sure the hardcoded list is up to date.

Reply:
(from offline discussion) it's necessary to make sure the run lists all have the same set of names, will add the test!

Reply:
Thanks!